packages/camera/lib/src/common/camera_mixins.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'camera_channel.dart';
+
+mixin NativeMethodCallHandler {
```

Comment:
The developer won't have access to the mixin, only their variables/methods. I created these because many classes needed them on Android and iOS, but I couldn't find a way to make them private and separate their libraries. I may just remove them from the `common` directory.